.github/workflows/pythonpublish.yml

```diff
@@ -17,6 +17,13 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.x"
+
+      - name: Verify version
+        run: |
+          if [[ "$(python3 setup.py --version)" != "${{ github.event.release.tag_name }}" ]]; then
+            exit 1
+          fi
```

Comment:
Nice little one 👍

Reply:
Should do the same...

```suggestion
      - name: Verify version
        run: |
          [[ "$(python3 setup.py --version)" != "${{ github.event.release.tag_name }}" ]] ```

Reply:
Neat, then this should work as well?
```suggestion
      - name: Verify version
        run: [[ "$(python3 setup.py --version)" != "${{ github.event.release.tag_name }}" ]]
```

Reply:
Yup that works too

Reply:
It did not :(
<https://github.com/NabuCasa/hass-nabucasa/pull/456>